azure-mgmt-batchai/azure/mgmt/batchai/models/unmanaged_file_system_reference.py

```diff
@@ -15,7 +19,7 @@ class UnmanagedFileSystemReference(Model):
     :type mount_command: str
     :param relative_mount_path: Specifies the relative path on the compute
      cluster node where the file system will be mounted. Note that all file
-     shares will be mounted under $AZ_BATCHAI_MOUNT_ROOT location.
+     shares will be mounted under $AZ_LEARNING_MOUNT_ROOT location.
```

Comment:
@AlexanderYukhanov why is that?

Reply:
Don't worry, it's because i submitted new swagger with renamed environment variables, but it's not merged yet. When it will be merged we will get AZ_BATCHAI back